components/studio/libexec/hab-studio-type-default.sh

```diff
@@ -219,7 +219,7 @@ case "\${HAB_STUDIO_SUP:-}" in
     # If false, we don't run the Supervisor
     ;;
   *)
-    sup-run "\${HAB_STUDIO_SUP:-}"
+    sup-run \${HAB_STUDIO_SUP:-}
```

Comment:
For these without an explicit shellcheck disable, I think a comment would be good so we don't repeat the past

Reply:
good point